docs/concepts/accounts-and-keys.md

```diff
@@ -82,7 +89,7 @@ secp256k1-keys/
 0 directories, 3 files
 ```
 
-The public-key-hex for `Secp256k1` keys starts with 02 and is 68 bytes long:
+The public-key-hex for `Secp256k1` keys starts with 02 and is 68 characters / 33  bytes long:
```

Comment:
@devendran-m I believe 68 means 34 bytes, simply divided by two

Reply:
Thanks, @gRoussac , and sorry for the oversight. I've addressed it now.